src/data/selector.ts

```diff
@@ -88,6 +89,13 @@ export interface ConfigEntrySelector {
   } | null;
 }
 
+export interface ConstantSelector {
+  constant: {
+    value?: string | number | boolean;
+    label?: string;
```

Comment:
Should we support translating the label, like we do with the select selector options?

Reply:
I can be nice in a future PR. As services are not translated yet, we don't need it immediately.

Reply:
Yes.

What format do we want ? I propose this one.

```json
"selector": {
  "white": {
    "value": "Enabled"
  }
}
```

Reply:
That seems reasonable 👍